lib/rrule/context.rb

```diff
@@ -17,14 +19,14 @@ def rebuild(year, month)
         possible_date_ranges = []
         if options[:freq] == 'YEARLY'
           if options[:bymonth]
-            options[:bymonth].each do |month|
-              possible_date_ranges.push(elapsed_days_in_year_by_month[(month - 1)..(month)])
+            options[:bymonth].each do |mon|
```

Comment:
What cop enforces this? I'm pretty opposed to any style rules that pushes us towards _less_ descriptive variable names.

Reply:
`month` shadows the outer variable `month`.